about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" /> 
+    <title>About:Capstone-I-Project</title>
+</head>
+<body>
+    <nav id="Mobile" class="Mobile-menu">
+        <div class="humburger">
+            <i class="fa-solid fa-bars"></i>
+        </div>
+    </nav>
+    <header class="first-bar">
+            <div><a href="#"><i class="fa-brands fa-facebook-f"></i></a></div>
+            <div><a href="#"><i class="fa-brands fa-twitter"></i></a></div> 
+            <div><a href="#">English</a></div>
+            <div><a href="#">My Page</a></div>
+            <div><a href="#">Logout</a></div>
+    </header>
+    <nav class="seconde-bar">
+            <a class="Logo" href="index.html"><img src="images/logo44.png" alt="logo-website" ></a>
+        <div class="menu">
+            <div class="closer"><i class="fa-solid fa-xmark"></i></div>
+            <div class="tags"><a id="edx" href="index.html">Home</a></div>
+            <div class="tags"><a href="#Program-cards">Program</a></div>
+            <div class="tags"><a href="#">Join</a></div>
+            <div class="tags"><a href="#partners">Sponsor</a></div>
+            <div class="tags"><a href="#post-card">News</a></div>
+            <div class="tags"><a id="efx" href="about.html">About</a></div>
+        </div>
+    </nav>
```

Comment:
- [ ] The top dark bar is part of the navbar. Kindly fix that.

Reply:
@leonardodiasb  Thank you for reviewing my project, I will make the necessary corrections as suggested